src/common/vector_operations/vector_copy.cpp

```diff
@@ -244,4 +239,4 @@ void VectorOperations::Copy(const Vector &source, Vector &target, idx_t source_c
 	}
 }
 
-} // namespace duckdb
+} // namespace duckdb
```

Comment:
What's the tidy implication here? I ask bc I have our `.editorconfig` set up to finish files with NL.